src/Site_WP_Command.php

```diff
@@ -242,6 +242,17 @@ public function create( $args, $assoc_args ) {
 		EE\Utils\delem_log( 'site create end' );
 	}
 
+	/**
+	 * Creates database user for a site
+	 *
+	 * @param string $site_url URL of site
+	 *
+	 * @return string Generated db user
+	 */
+	private function create_site_db_user( string $site_url ) : string {
+		return $site_url . '-' . \EE\Utils\random_password( 6 );
```

Comment:
@kirtangajjar Can you please add some limitation to the site_url string length?
We are currently using mariadb and it has database user name limit to 80 char.
> Users have a maximum length of 80 characters.

ref: https://mariadb.com/kb/en/library/identifier-names/#maximum-length

and here our `site_url` can have more then 80 char - considering the site url might be like `example.com` `test.example.com` `test.example.com/test/` ...

Reply:
Let me know if current limit isn't good enough